Modules/_io/iobase.c

```diff
@@ -962,11 +962,9 @@ static PyObject *
 _io__RawIOBase_readall_impl(PyObject *self)
 /*[clinic end generated code: output=1987b9ce929425a0 input=688874141213622a]*/
 {
-    int r;
-    PyObject *chunks = PyList_New(0);
-    PyObject *result;
+    PyBytesWriter *writer = PyBytesWriter_Create(0);
 
-    if (chunks == NULL)
```

Comment:
nit: When touching code without curlies it's good to add them per PEP 7

Reply:
b8a7f89a04f1dd8bfa43a29ff3b9ec2b8c190f33